### DISCUSSION TITLE
Why is the Java segment shown at root?

### DISCUSSION THREAD

**Discussion by TheOnlyTails** [JanDeDobbeleer/oh-my-posh#1218] [Troubleshoot]
As you can see, I don't have any file here that would cause the Java segment to be shown, yet it is.
![image](https://user-images.githubusercontent.com/65342367/141499479-ef6cb0db-6d67-4ab4-aa1f-ee56fe8d95e4.png)

My theme: https://gist.github.com/3249cad2e680078bedce3ec749775d87

**Comment by JanDeDobbeleer** [comment]
@TheOnlyTails these are all the files we're triggered by:

```json
"pom.xml",
"build.gradle.kts",
"build.sbt",
".java-version",
".deps.edn",
"project.clj",
"build.boot",
"*.java",
"*.class",
"*.gradle",
"*.jar",
"*.clj",
"*.cljc",
```

If you want to disable the `$HOME` folder, you can also set `"home_enabled": false`.

**Reply by TheOnlyTails** [reply]
I've tried adding `"display_mode": "files"` and `"home_enabled": false`, but it didn't work.

**Reply by JanDeDobbeleer** [reply]
I'll have to debug this. Shouldn't happen though, the logic is identical to any other language.
